### DISCUSSION TITLE
Is anyone working on this topic: customers' persona extraction from tons of service recordings?

### DISCUSSION THREAD

**Discussion by yanshili645** [community/community#110358] [Programming Help]
My team are recently working on a project to make a AI chatbot to coach new customer service representatives. We are expecting that the chatbot can imitate the real customer to do inquiry with the new-hired representatives (only based on LLM and vertival model without any process designed behind).

Here comes the pain point:  **the persona of the customer mimic by the chatbot is not stable and lacks of diversity.** 

If you have relevant experience or solution, please contact me. You might be the consultant or co-founder we are looking for in near future.

**Comment by queenofcorgis** [comment]
Hi @yanshili645 , we're happy you're here 😀

However, we currently do not allow job listings or seeking on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
